From 4087346ee7036b8d5966c593e98c99bbd6a1ac16 Mon Sep 17 00:00:00 2001
From: Peter Lundkvist <peter.lundkvist@hectronic.se>
Date: Thu, 16 Apr 2026 11:29:40 +0200
Subject: [PATCH 1/5] imx8mq: set bus frequency scaling default to disabled
Upstream-Status: Pending

This could be changed later by setting
/sys/bus/platform/drivers/imx_busfreq/busfreq/enable to 1 or 0.
---
 drivers/soc/imx/busfreq-imx8mq.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/soc/imx/busfreq-imx8mq.c b/drivers/soc/imx/busfreq-imx8mq.c
index 5e36962285ef..fa54e5c8b04c 100644
--- a/drivers/soc/imx/busfreq-imx8mq.c
+++ b/drivers/soc/imx/busfreq-imx8mq.c
@@ -610,7 +610,7 @@ static int busfreq_probe(struct platform_device *pdev)
 	audio_bus_freq_mode = 0;
 	cur_bus_freq_mode = BUS_FREQ_HIGH;
 
-	bus_freq_scaling_is_active = 1;
+	bus_freq_scaling_is_active = 0;
 	bus_freq_scaling_initialized = 1;
 
 	INIT_DELAYED_WORK(&low_bus_freq_handler, reduce_bus_freq_handler);
-- 
2.53.0

